reference-implementation/to-upstream-wpts/writable-streams/close.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 if (self.importScripts) {
-  self.importScripts('/resources/testharness.js');
+  self.importScripts('/resources/testharness.js', '..resources/test-utils.js');
```

Comment:
I thought that the multiple-argument form permitted the requests to proceed in the parallel, but it appears not.

Changed.